nosql/JobListing.json

```diff
@@ -0,0 +1,17 @@
+{
+  "jobID":"Integer",
+  "industryType":"String",
+  "employmentType":"String",
+  "jobFunctions":"String",
+  "positionName":"String",
+  "professionLevel":"String",
+  "companyID":"Number",
+  "companyName":"String",
+  "companyLocation":"String",
+  "compnayPicture":"URL/String",
+  "relatedConnections":[{},{}],
```

Comment:
since the related connections are users/ ids. it's a list of numbers --> ["Number"]